subsys/bluetooth/audio/bap_unicast_client.c

```diff
@@ -4467,14 +4467,13 @@ static void unicast_client_disconnected(struct bt_conn *conn, uint8_t reason)
 	unicast_client_ep_reset(conn, reason);
 }
 
-static struct bt_conn_cb conn_cbs = {
+BT_CONN_CB_DEFINE(conn_cbs) = {
```

Comment:
Could you separate the new Kconfig option and the changes to the calls of `bt_conn_cb_register` into separate commits? Also splitting host from audio?